server/app/Console/Commands/DatabaseRebuild.php

```diff
@@ -3,28 +3,32 @@
 namespace App\Console\Commands;
 
 use Illuminate\Console\Command;
+use Symfony\Component\Console\Input\InputOption;
 
 class DatabaseRebuild extends Command
 {
     /**
-     * The console command name.
+     * The name and signature of the console command.
      *
      * @var string
      */
-    protected $signature = 'db:rebuild {--database=mysql}';
+    protected $signature = 'db:rebuild {--database=}';
```

Comment:
It does remove the default as defaulting to a database called `mysql` seemed like a bad idea, but I see now that it doesn't specify the actual database to use, but rather which application database connection to use, so it's just badly named. I see that Laravel's commands use the same terminology though, so I guess we'll keep it. I'll add the default back in.